README.md

```diff
@@ -1 +1,72 @@
-# rhbk-quickstarts
\ No newline at end of file
+# <span>RHBK</span> Quickstarts
+
+<span>Red Hat build of Keycloak (RHBK)</span> is an Open Source Identity and Access Management solution for modern Applications and Services.
```

Comment:
I don't think we should use RHBK as I don't think we are really permitted to use the abbreviation by RH marketing. Can you replace RHBK throughout with Red Hat build of Keycloak?